plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -606,6 +607,31 @@ public void testBigQueryExternalTable()
         }
     }
 
+    @Test
+    public void testQueryLabel()
+    {
+        String materializedView = "test_query_label" + randomNameSuffix();
```

Comment:
is there a specific reason to use an MV? Add a comment or simplify.

Reply:
Yes, labels are added to jobs and jobs are used only when querying non-storage-backed relations (i.e. views)